torch/distributed/rpc.py

```diff
@@ -62,10 +73,19 @@ def _init_rpc(name, backend=RpcBackend.PROCESS_GROUP):
         raise RuntimeError("RPC is already initialized")
 
     if backend == RpcBackend.PROCESS_GROUP:
-        from .distributed_c10d import _get_default_group
+        from .distributed_c10d import _get_default_group, init_process_group
+
+        init_process_group(
```

Comment:
Any reason for putting `init_process_group` in `init_rpc`? Would it be better if we keep them separate as what we did for `DistributedDataParallel`?